src/main/kotlin/tech/harmonysoft/oss/gradle/dist/BuildCustomGradleDistributionTask.kt

```diff
@@ -73,7 +104,7 @@ abstract class BuildCustomGradleDistributionTask : DefaultTask() {
     private fun doGetBaseGradleDistribution(extension: CustomGradleDistConfig): File {
         val gradleBaseName = "gradle-${extension.gradleVersion.get()}"
         val gradleZip = "$gradleBaseName-${extension.gradleDistributionType.get()}.zip"
-        val baseGradleArchive = project.layout.buildDirectory.file("download/$gradleZip").get().asFile
+        val baseGradleArchive = downloadRootDir.map { it.dir(gradleZip) }.get().asFile
```

Comment:
why do we use `dir()` here? `file()` is more natural

Reply:
You're right, apparently I was too hasty. I fixed it.